tests/api_converters_wrappers.js

```diff
@@ -0,0 +1,150 @@
+var tape = require("tape");
+
+var protobuf  = require("..");
+
+tape.test("converters", function(test) {
+
+    protobuf.load("tests/data/convert_wrappers.proto", function(err, root) {
+        if (err)
+            return test.fail(err.message);
+
+        var Message = root.lookup("TestMessage");
+
+        test.test(test.name + " - Message.fromObject - wrappers happy path", function(test) {
+
+            var obj = {
+              id: 1,
+              simpleField: "string value",
+              wrappedField: "string value",
+              simpleId: 1,
+              wrappedId: 1,
+              simpleBool: true,
+              wrappedBool: true,
+              longField: "432456523629741056"
```

Comment:
@dkrol-mix i think we could remove `longField` from these tests. I added to our prototype for validating our specific use-case of 64-bits longs.